README.md

```diff
@@ -1,16 +1,16 @@
-# Geocortex Web Component Incubator
+# VertiGIS Studio Web Component Incubator
```

Comment:
Thanks for making these changes. I think though that they could conflict with Scott's other outstanding PR for the name change (hopefully they are identical, but you never know). I think that it might be best to leave existing instances of Geocortex alone and only fix the net new code.

Reply:
Text changes in root readme and mapillary reverted